content/docs/esc/integrations/dynamic-secrets/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Dynamic Secrets
```

Comment:
Yeah, again -- it's really not about "dynamic secrets", it's about "fetching and using secrets from  other providers".

Reply:
Adjusted the H1 to be more descriptive